dry-run/symphonik/SymphonIK.py

```diff
@@ -0,0 +1,227 @@
+
+logger = logging.getLogger(__name__)
+
+
+# Obelisk_Ichimoku_Slow v1.3 - 2021-04-20
+#
+# by Obelisk
+# https://github.com/brookmiles/
+#
+# 1.3 increase return without too much additional drawdown
+#  - add ema entry guards
+#  - remove cloud top exit signal from 1.2
+#
+# The point of this strategy is to buy and hold an up trend as long as possible.
+# If you are tempted to add ROI or trailing stops, you will need to make other modifications as well.
+#
+# This strategy can optionally be backtested at 5m or 1m to validate roi/trailing stop behaviour (there isn't any).
+#
+# WARNING
+#
+# Do not use stoploss_on_exchange or the bot may trigger emergencysell when it
+# fails to place the stoploss.
+#
+# WARNING
+#
+# This strat will buy into ongoing trends, so pay attention to timing when you launch it.
+# If the market is peaking then you may end up buying into trends that are just about to end.
+#
+#
+# Contributions:
+# JimmyNixx - SSL Channel confirmation
+#
+# Backtested with pairlist generated with:
+#     "pairlists": [
+#         {
+#             "method": "VolumePairList",
+#             "number_assets": 25,
+#             "sort_key": "quoteVolume",
+#             "refresh_period": 1800
+#         },
+#         {"method": "AgeFilter", "min_days_listed": 10},
+#         {"method": "PrecisionFilter"},
+#         {"method": "PriceFilter", "low_price_ratio": 0.001},
+#         {
+#             "method": "RangeStabilityFilter",
+#             "lookback_days": 3,
+#             "min_rate_of_change": 0.1,
+#             "refresh_period": 1440
+#         },
+#     ],
+
+def ssl_atr(dataframe, length=7):
+    df = dataframe.copy()
+    df['smaHigh'] = df['high'].rolling(length).mean() + df['atr']
+    df['smaLow'] = df['low'].rolling(length).mean() - df['atr']
+    df['hlv'] = np.where(df['close'] > df['smaHigh'], 1,
+                         np.where(df['close'] < df['smaLow'], -1, np.NAN))
+    df['hlv'] = df['hlv'].ffill()
+    df['sslDown'] = np.where(df['hlv'] < 0, df['smaHigh'], df['smaLow'])
+    df['sslUp'] = np.where(df['hlv'] < 0, df['smaLow'], df['smaHigh'])
+    return df['sslDown'], df['sslUp']
+
+
+def create_ichimoku(dataframe, conversion_line_period, displacement, base_line_periods, laggin_span):
+    ichimoku = ftt.ichimoku(dataframe,
+                            conversion_line_period=conversion_line_period,
+                            base_line_periods=base_line_periods,
+                            laggin_span=laggin_span,
+                            displacement=displacement
+                            )
+    dataframe[f'tenkan_sen_{conversion_line_period}'] = ichimoku['tenkan_sen']
+    dataframe[f'kijun_sen_{conversion_line_period}'] = ichimoku['kijun_sen']
+    dataframe[f'senkou_a_{conversion_line_period}'] = ichimoku['senkou_span_a']
+    dataframe[f'senkou_b_{conversion_line_period}'] = ichimoku['senkou_span_b']
+
+
+class SymphonIK(IStrategy):
+    # Optimal timeframe for the strategy
+    timeframe = '1m'
```

Comment:
Do we want to act every minute or act every 12h ?

Reply:
Every 1m